extmod/moduselect.c

```diff
@@ -150,7 +150,7 @@ STATIC mp_obj_t select_select(size_t n_args, const mp_obj_t *args) {
             mp_map_deinit(&poll_map);
             return mp_obj_new_tuple(3, list_array);
         }
-        MICROPY_EVENT_POLL_HOOK
+        RUN_BACKGROUND_TASKS;
```

Comment:
These changes look to be async related.

Reply:
No, this was a git mess-up. I added only certain files, but must have missed some.